cmd/destroy.go

```diff
@@ -81,6 +89,7 @@ var destroyCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(destroyCmd)
+	destroyCmd.Flags().BoolVarP(&cleanup, "cleanup", "", false, "delete config artefacts")
```

Comment:
we know what config artefacts are, likely users dont
maybe give a simplified description of `delete lab directory`?